src/main/java/io/dropwizard/revolver/retry/ValidResponseFilter.java

```diff
@@ -27,6 +29,9 @@ private boolean validateResponse(Object object) {
             return false;
         }
         Response response = (Response)object;
+        if(response.code() >= 500){
```

Comment:
500 should not be retried. we should be retrying 503 and 504 only. 500 might be a real error.